modular_splurt/code/modules/jobs/job_types/_job_alt_titles.dm

```diff
@@ -238,38 +309,44 @@
 	var/list/extra_titles = list(
 		"Therapist",
 		"Psychiatrist",
-		"Hypnotist",
-		"Hypnosis Expert",
 		"Hypnotherapist",
-		"Sex Educator",
-		"Rental Mommy",
-		"Rental Daddy",
-		"Psycholo-Slut"
+		"Counselor",
```

Comment:
```suggestion
		"Hypnotist",
		"Hypnosis Expert",
		"Sex Educator",
		"Rental Mommy",
		"Rental Daddy",
		"Psycholo-Slut",
		"Counselor",
```

Reply:
There are three hypnosis-related titles, it's redundant to have all of them. Additionally, I deleted all the medical slut titles because frankly, they're extremely bland. I'd love to have creative lewd titles, but these are just kinda shit. also, only 19 people wanted 'em.

Reply:
Plus, rental mommy and daddy don't need the "Rental" in front. I'll add back sex educator and psycholo-slut